client/my-sites/site-settings/seo-settings/form.jsx

```diff
@@ -413,8 +416,8 @@ export const SeoForm = React.createClass( {
 		};
 
 		const nudgeTitle = jetpack
-			? this.translate( 'Enable SEO Tools Features by Upgrading to Jetpack Professional' )
-			: this.translate( 'Enable SEO Tools Features by Upgrading to the Business Plan' );
+			? this.translate( 'Enable SEO Tools features by upgrading to Jetpack Professional' )
+			: this.translate( 'Enable SEO Tools features by upgrading to the Business Plan' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Enable SEO Tools Features by Upgrading to the Business Plan' )` *ES Score: 18.09*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/site-settings/11176-seo-banner-overload)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).